core/modules/czar/python/proxy.py

```diff
@@ -50,9 +63,11 @@ class Lock:
     requests when it is waiting on mysql queries that it knowingly
     initiates in its C-layer, whereas it treats its Lua plugin code as a
     single-threaded black-box."""
-    createTmpl = "CREATE TABLE IF NOT EXISTS %s (chunkId INT, code SMALLINT, message CHAR(255), timeStamp FLOAT) ENGINE=MEMORY;"
+    createTmpl = "CREATE TABLE IF NOT EXISTS %s " \
+                 "(chunkId INT, code SMALLINT, message CHAR(255), " \
+                 "severity ENUM ('DEBUG', 'INFO', 'ERROR'), timeStamp FLOAT) ENGINE=MEMORY;"
```

Comment:
Only "INFO" and "ERROR" are useful for message. lua log "INFO" and send "ERROR" to client. So I removed "DEBUG" for now.